acme/acme/crypto_util.py

```diff
@@ -579,3 +580,34 @@ def _dump_cert(cert: Union[jose.ComparableX509, crypto.X509]) -> bytes:
     # assumes that OpenSSL.crypto.dump_certificate includes ending
     # newline character
     return b"".join(_dump_cert(cert) for cert in chain)
+
+def ariCertIdent(cert: crypto.X509) -> str:
```

Comment:
Can we make this take a cryptography cert directly?

Reply:
I'll change this function to use cryptography cert:
looking at it acme/_internal/tests/test_util loads certificate as pyopenssl: will need additional PR about them

Reply:
`jose.ComparableX509` is being dropped in the next major release; `acme` will break and require a PR to remove all that code - this is already known. 
 The certbot team preferred a hard breaking change to alternatives.  Just expect anything in jose and acme that uses ComparableX509 to accept/return a cryptography.x509 object instead.